src/interfaces/Command.ts

```diff
@@ -0,0 +1,8 @@
+import { CommandInteraction } from 'discord.js';
+import { RESTPostAPIChatInputApplicationCommandsJSONBody } from 'discord-api-types/v10';
+
+export interface Command {
+	commandStrucure: RESTPostAPIChatInputApplicationCommandsJSONBody;
```

Comment:
Typo here and in all references.

`strucure` -> `structure`

```suggestion
	commandStructure: RESTPostAPIChatInputApplicationCommandsJSONBody;
```

Reply:
There is always a problem to me when I write this word LOL. Will fix in the next commit.